### ISSUE TITLE
some attributes allow different ways to write which may be confusing

### ISSUE THREAD

**Issue by sopos** [psss/tmt#683]
Typical example of this is `require` attribute. It may be written as plain on-line text, meaning one item list or as a list of more values. The same applies to the steps in plans.

This may lead to a confusion where one is not sure what can be achieved in which attribute.

The minimal way would be to consistently use lists `-` is all examples. It may look more complicated but at the same time it will increase a level of certainty.

More ultimate solution would be to forbid this ambiguity in next _tmt_ data structure version.

**Comment by psss** [comment]
Yeah, sometime might be confusing. On the other hand it provides a way for concise config. Seems that the explicit documentation helps here a lot, for example:

* https://tmt.readthedocs.io/en/stable/spec/tests.html#require

I'm not sure it would be worth it now introducing a backward-incompatible change. Was not raised for quite some time. Closing for now, we can reopen later.
